data/brands/shop/supermarket.json

```diff
@@ -4093,14 +4093,62 @@
       }
     },
     {
-      "displayName": "Masymas",
-      "id": "masymas-0513f1",
-      "locationSet": {"include": ["es"]},
+      "displayName": "masymas (Juan Fornés Fornés S.A.)",
+      "locationSet": { "include": [[-0.7197, 39.4717, 60],[-0.3, 38.6983, 60],[-1.041389, 38.207222, 60],[-1.6023, 37.7126, 50],[-0.8792, 37.6237, 20],[0.0096, 40.3, 60]] },
```

Comment:
I'm confused that why so much of location are defined by a series of circle……It's different for other people's understand or modification. Maybe you can add a `note` key?

Reply:
I know that is a bit weird. This is from the [website](http://[www.masymas.es/layout/portada_geolocalizada.php?Idioma=en](http://www.masymas.es/layout/portada_geolocalizada.php?Idioma=en)) BRAND SHARED BY 4 DIFFERENT COMPANIES, WITH TOTALLY INDEPENDENT MANAGEMENT, DISTRIBUTED IN DIFFERENT STATES.

They only share the name and the own brand of supermarket products (in Spanish white label)